docs/source/guide/index.md

```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
+
+**Data annotation**
+Data annotation is the categorization and labeling of data for AI applications. It helps machines to learn the certain patterns and correlate the results, and the use the data sets to recognize the similar patterns in future to predict the results.
+
+**Data labeling**
+Data labeling is the process of identifying raw data (images, text files, videos, and so on) and adding one or more meaningful and informative labels to provide context so that a machine learning model can learn from it.
+
+**Data parsing**
+Data parsing is a method of converting data from one format to another format. For example, converting from HTML to JSON.
+
+**Dialog processing**
+Dialog processing or interactive document creation is the complementary method to batch processing and background processing. It is defined by the active user who creates all or part of the document in the system editor.
+
+**Docker**
+Docker is a software platform that allows you to build, test, and deploy applications quickly.
+
+**Document classification**
+Document classification is the act of labeling or tagging documents using categories, depending on their content.
+
+**Dataset**
+A dataset (or data set) is a collection of data comprised of individual items, or labeling tasks that you import into Label Studio.
+
+**Emotion recognition**
+Emotion recognition is the process of identifying human emotions.
+
+**Entity recognition**
+Entity recognition is a new technology that can enable investigators and document review managers to get to the pertinent data faster.
+
+**Human labeling**
+Human labeling enables you to label images in your training dataset at scale.
+
+**Image classification**
+Image classification is the process of categorizing and labeling groups of pixels or vectors within an image based on specific rules.
+
+**Image labeling**
+Image labeling is the process of identifying and marking various details in an image. It is useful when automating the process of generating metadata or making recommendations to users based on details in their images.
+
+**Image segmentation**
+Image segmentation is the process of assigning a label to every pixel in an image such that pixels with the same label share certain characteristics.
+
+**Image tagging**
+Image tagging is the process of labeling images based on figures within a certain picture. It enables the classification of visuals through the use of tags and labels.
+
+**Inventory tracking**
+Inventory tracking system allows you to label exact products by given brand names illustrated by relevant sample photo.
+
+**Label**
+A label is a key-value pair, stored as a string. You can specify multiple labels for an object, but each key must be unique within an object.
```

Comment:
It's weird description for label term, better to remove label from here and put it into Label Studio Terms section